utils/connects.stanza

```diff
@@ -85,3 +85,21 @@ public defn connect-cp2105 (usb:JITXObject, uart:JITXObject) -> Instance  :
     net (uart.dtr, xcvr.e-uart.dtr)
     net (uart.rts, xcvr.e-uart.rts)
     xcvr
+
+; Connects a SPI controller to 1 or more peripherals, pulls gpio for peripherals[1..n] for cs
+public defn connect-spi (controller:JITXObject, peripherals:Tuple<JITXObject>) :
```

Comment:
It would be great to create a unit tests for this function

Reply:
`fatal("Your message") when empty?(peripherals)`

Reply:
added usb-accel.stanza to test spi bundles

Reply:
created tests/test-bundle-connects.stanza to test function